src/webots/vrml/WbProtoModel.hpp

```diff
@@ -145,9 +147,11 @@ class WbProtoModel : public QObject {
   QStringList mTags;
   QString mTemplateLanguage;
 
+  QMap<QString, QString> mParameterAliases;  // tracks the connections between an exposed parameter and its internal counterpart
+
   ~WbProtoModel();  // called from unref()
-  void verifyAliasing(WbNode *root, WbTokenizer *tokenizer) const;
-  void verifyNodeAliasing(WbNode *node, WbFieldModel *param, WbTokenizer *tokenizer, bool searchInParameters, bool &ok) const;
+  void verifyAliasing(WbNode *root, WbTokenizer *tokenizer);
+  void verifyNodeAliasing(WbNode *node, WbFieldModel *param, WbTokenizer *tokenizer, bool searchInParameters, bool &ok);
```

Comment:
I would rename these functions to `setupAliasing` and `setupNodeAliasing` as the `verify` prefix is a bit misleading (they also construct the parameter alias map).